core/node/libp2p/rcmgr_defaults.go

```diff
@@ -39,9 +39,10 @@ func adjustedDefaultLimits(cfg config.SwarmConfig) rcmgr.DefaultLimitConfig {
 		maxconns := cfg.ConnMgr.HighWater
 		if 2*maxconns > defaultLimits.SystemBaseLimit.ConnsInbound {
 			// adjust conns to 2x to allow for two conns per peer (TCP+QUIC)
-			defaultLimits.SystemBaseLimit.ConnsInbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.ConnsOutbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.Conns = logScale(4 * maxconns)
+			// and 2x again to allow for burst of conns during accelerated DHT client rt refresh
+			defaultLimits.SystemBaseLimit.ConnsInbound = logScale(4 * maxconns)
+			defaultLimits.SystemBaseLimit.ConnsOutbound = logScale(4 * maxconns)
+			defaultLimits.SystemBaseLimit.Conns = logScale(8 * maxconns)
```

Comment:
I'm not sure that only doing this in the context of  `2*HighWater > defaultLimits.SystemBaseLimit.ConnsInbound` makes sense. Even if your HighWater is lower (e.g. 100 connections) you might still want to allow for spikiness when your application demands it (e.g. for the accelerated DHT client routing table refreshes).

Reply:
Do you have a suggestion here?  I'm mostly focused on making sure this works for the default scenario, not for folks tweaking connmgr or rcmgr settings.